src/Controllers/ContentController.php

```diff
@@ -835,6 +855,10 @@ public function filterContents(Request $request)
         ModeDecoratorBase::$decorationMode = DecoratorInterface::DECORATION_MODE_MAXIMUM;
         $collectionForDecoration = Decorator::decorate($collectionForDecoration, 'content');
 
+        if($request->get('group_by', false)){
+            $collectionForDecoration =  Decorator::decorate($collectionForDecoration, 'group');
+        }
+
         return ResponseService::catalogue($results, $request);
```

Comment:
Should the `collectionForDecoration` be used here instead of the `$results`? Unless I'm missing something, the whole decoration process won't have any impact on the returned response.

Reply:
The collection from $results in decorated